### PR TITLE
[Daily] Fix daily update composer update

### PR DESCRIPTION
**Pull request by samsonasik**
Update `"phpstan/phpstan-doctrine` to `^1.0` and `symplify` packages to `^10.0`.

solve https://github.com/rectorphp/getrector.org/runs/4122986149?check_suite_focus=true#step:4:12

### CODE REVIEW

File: packages/Demo/Entity/RectorRun.php
Comment (samsonasik) [inline packages/Demo/Entity/RectorRun.php:21]: @TomasVotruba I updated to use the `PackageBuilder` StringFormatConverter as `StringsConverter` seems functionality is in there and `SymplifyKernel'`s StringsConverter no longer exists in version `^10.0`

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: packages/Demo/Entity/RectorRun.php
Comment (TomasVotruba) [reply packages/Demo/Entity/RectorRun.php:21]: Yes, thank you. That was intended only for internal Kernel naming, not sure what is doing here.

File: packages/Demo/Entity/RectorRun.php
Comment (TomasVotruba) [inline packages/Demo/Entity/RectorRun.php:24]: This will probably fail in Doctrine.

Comment (samsonasik) [reply packages/Demo/Entity/RectorRun.php]: Reverted https://github.com/rectorphp/getrector.org/commit/896cd5908a50fca9f269db38e6b80cd517e9b6ff

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

File: rector.php
Comment (samsonasik) [inline rector.php:44]: @TomasVotruba the `DemoFile.php` is on root `./data' directory, do you think it can be removed from the skip option list as the file is not exists in the packages dir?

Comment (samsonasik) [reply rector.php]: I removed it from the skip config list of ecs, phpstan, and rector.php as file moved to root `data` directory https://github.com/rectorphp/getrector.org/pull/441/commits/55dddf42c495b3106f29d98b5510c9f3390922c4

Comment (TomasVotruba) [reply rector.php]: Ok :+1:

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Feel free to merge :+1:

File: composer.lock
Comment (samsonasik) [inline composer.lock:3557]: by requiring phpstan ^1.0 with minimum-stability dev, it seems we will have `"rector/rector": "dev-main"` in the composer.lock before next rector release. The site is using php ^8.0, so I think it is fine so we can catch the bug early.

File: rector.php
Comment (samsonasik) [reply rector.php]: @TomasVotruba it seems there is 500 error at demo page now after merge

Screen Shot 2021-11-07 at 18 27 37

I will revert it to see if that related to it.

**Comment by TomasVotruba** [comment]
@samsonasik This whole PR should be reverted, before we figure out what happened.  Including dependencies
Now the demo page is down and people can't report issues or try Rector.

**Comment by TomasVotruba** [comment]
The partial manaul revert of this PR https://github.com/rectorphp/getrector.org/commit/241490948f363803578b9a054c22e6fe17238e6b makes it hard to revert easily.

I'll have to revert last 2 commits with force push. Let's see if it helps

**Comment by samsonasik** [comment]
@TomasVotruba ok, the demo page seems working ok now. Thank you 👍

**Comment by TomasVotruba** [comment]
I see it too... https://getrector.org/demo/1ec3fd64-aea9-6504-9729-3dcbb9828cf0

Next time please revert the PR immediately to last working commit hash, if there is any 500 on the website. Then we can fix it locally in peace, instead of stressing with broken website.

Thanks :+1: 

IMO it will be related with Symfony 5.4 used here and Symfony 5.3 used in Rector. I had some simliar issues on rector/rector. We should wait for release of Symfony 5.4 stable and then use it here and in Rector itself.

**Pull request merged**
